src/components/common/header/Header.js

```diff
@@ -13,7 +17,22 @@ export default () => {
 
   return (
     <Fragment>
-      <AuthentictedHeader />
+      {isAuthenticated ? (<AuthentictedHeader />) : (<UnauthenticatedHeader />)}
```

Comment:
Did you really have to create two components for each state?

Reply:
Not really, this happened when I was trying to create both components separately and we just ran with it because we could manage the contents in the navbar more easily and neatly